utils/build.ps1

```diff
@@ -26,9 +26,6 @@ The path to a directory that mimics a file system image root,
 under which "Library" and "Program Files" subdirectories will be created
 with the files installed by CMake.
 
-.PARAMETER BuildType
```

Comment:
Is the removal of debug builds related here? sccache should see the flag in the compiler invocation and realize that it's a different object being emitted.

Reply:
Yes, it is related.  The Swift build system is just not able to support this (sccache + PDBs don't work very well together, the other changes to use the embedded debug is related).  Once we upgrade to a reasonable version of CMake, we should be able to use the `CMAKE_MSVC_DEBUG_INFORMATION_FORMAT=Embedded` and get the build types to work again.

Reply:
hmm, okay :/